packages/test/test-end-to-end-tests/src/test/sharedStringLoading.spec.ts

```diff
@@ -90,10 +89,6 @@ describeNoCompat("SharedString", (getTestObjectProvider) => {
                     mockDs.connectToStorage = async () => {
                         const realStorage = await realDs.connectToStorage();
                         const mockstorage = Object.create(realStorage) as IDocumentStorageService;
-                        (mockstorage as any).policies = {
```

Comment:
May be it will fail with r11s/tinylicious as that flag is only used there?

Reply:
looks like @vladsud added this as part of the blob aggregation work:
https://github.com/microsoft/FluidFramework/pull/5032

Reply:
it wasn't part of the original test in the above commit

Reply:
You are right, I did not trace history all the way with renames.
I'm not sure why it was needed at the time, I think there were some issues with prefetcher, and I remember there were some changes in R11s stack around these policies, but nothing specific comes to mind.
Tests are passing, so I think it's safe to remove it.